### DISCUSSION TITLE
Issue with equation rendering and symbols

### DISCUSSION THREAD

**Discussion by felwind14** [dita-ot/dita-ot#4517] [Q&A]
Dear all,

Maybe an easy one for somebody but hard one for me.

I have equations on Oxygen already inserted however, when I try to render the equations do not appear on the pdf output but it does on html. Similarly it happens with any greek symbol like <html> &Sigma;

Could you give me hints how to solve these two issues? I am not sure if they are related.

Thank you so much

**Comment by raducoravu** [comment]
Are you using the DITA OT publishing engine bundled with Oxygen or a custom DITA OT for publishing? Does anything appear instead of the equation?

**Reply by felwind14** [reply]
It is a custom DITA OT, I am using oxygen just for authoring.

**Reply by raducoravu** [reply]
A default DITA OT distribution does not have default support for MathML equations, there is a long discussion and some advice here:
https://github.com/dita-ot/dita-ot/issues/1774
Oxygen's bundled DITA OT does have the proper plugins installed to support MathML equations in PDF output.

**Reply by stefan-jung** [reply]
@felwind14 Try this plugin https://github.com/super-structure/com.bentley.math-content

**Reply by chrispy-snps** [reply]
This reminds me that per the discussion in #1774, I need to look into how inline equation SVGs might be written out as on-disk SVGs during publishing...
